service/src/test/java/bio/terra/catalog/iam/UserTest.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.catalog.iam;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import bio.terra.common.exception.UnauthorizedException;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+
+class UserTest {
+
+  private final MockHttpServletRequest request = new MockHttpServletRequest();
+
+  private final User user = new User(request);
+
+  @Test
+  void getTokenOauth2() {
+    var token = "token";
+    request.addHeader(User.OAUTH2_ACCESS_TOKEN, token);
```

Comment:
This code is now gone, but to answer the question, there's no risk because JUnit creates a new class instance for each test method invocation. So each test method gets a different copy of the test class. (I seem to recall TestNG shares the test class instance across test methods, so maybe that's what you're thinking about here.)